GoogleForCreators/web-stories-wp
# Do not unnecessarily fetch stories/taxonomies

## Context



When testing #12626 but also otherwise, I noticed superfluous HTTP requests for fetching stories and taxonomies when not even visiting the dashboard itself.

## Summary



Do not unnecessarily fetch stories/taxonomies when directly visiting a sub page like the settings page.

## Relevant Technical Choices



## To-do



## User-facing changes



N/A

## Testing Instructions






This PR can be tested by following these steps:

1. Visit `/wp-admin/edit.php?post_type=web-story&page=stories-dashboard#/editor-settings` directly
2. See no HTTP requests for fetching stories and taxonomies
3. Switch to Dashboard
4. See stories & taxonomies filters loading


## Reviews

### Does this PR have a security-related impact?



No

### Does this PR change what data or activity we track or use?



No

### Does this PR have a legal-related impact?



No

## Checklist




---



Fixes #
